PartTwo/PartTwo/SocialMediaView.m

```diff
@@ -0,0 +1,29 @@
+//
+//  SocialMediaView.m
+//  PartTwo
+//
+//  Created by Christella on 9/18/15.
+//  Copyright © 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "SocialMediaView.h"
+#import "SocialMediaBackgroundProtocol.h"
+
+
+
+@implementation SocialMediaView
+
+
+
+#pragma mark - Colors Delegate
```

Comment:
These methods should go into `ViewController.m`. `ViewController` is the class that *conforms to the delegate*. In this case, `ViewController` would be the delegate of `SocialMediaView`.